docs/specs/api.md

```diff
@@ -59,6 +59,20 @@ The challenge is used in anonymous user and biometric authentication.
 
 This api is for signing up an new anonymous user in the Web SDK.
 
+For the `cookie` app,
+
+1. The api will check the cookie.
+1. If there is no logged in session, new user will be created. The server will return the `Set-Cookie` header and the result object will be an empty object.
+1. If the logged in user is an anonymous user, a success response will be returned. No new cookie header will be return and the result object will be an empty object.
+1. If the logged in user is a normal user, an error will be returned.
+
+For the `refresh_token` app,
+
+1. If the user has logged in, `refresh_token` will be included in the request.
+1. If there is no logged in session, new user will be created. `refresh_token` and `access_token` will be issued in the result object.
+1. If the logged in user is an anonymous user, a success response will be returned. New `access_token` will be issued in the result object.
```

Comment:
I am thinking for both cookie and refresh token case, should we extend the lifetime of the session if the developer calls authenticateAnonymously again. If yes, this behavior should be stated here.

Reply:
I think it is not very useful since if developer use the api correctly, they should only call `authenticateAnonymously` when there is no logged in session. What do you think?